FrameworkSystem/DB/InstalledComponentsDB.py

```diff
@@ -0,0 +1,758 @@
+"""
+Classes and functions for easier management of the InstalledComponents database
+"""
+
+__RCSID__ = "$Id$"
+
+import datetime
+from sqlalchemy import *
```

Comment:
import * should be forbidden for many different reasons. Explicit import of symbols actually used in the code must be done.